app/views/clients/clients.html

```diff
@@ -3,10 +3,16 @@
         <li class="active">{{'label.anchor.clients' | translate}}</li>
     </ul>
     <api-validate></api-validate>
-
-    <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="marginbottom0px form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
-    <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
-
+    <div class="col-sm-5">
+        <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
+    </div>
+        <form ng-submit="search()">
+            <div class="col-sm-4">
+            <input  data-ng-model="client.query" type="text" placeholder="Search Client with Name/ExternalId/Identifier/Acc.No" value="{{query}}" class="form-control"/>
+            </div>
+            <button id="query" type="search" class="btn btn-primary" has-permission='READ_CLIENT'>Search Client</button>
+            <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
+        </form>
     <table class="table">
```

Comment:
ok sir.

I will do it and I will send pull request again.

Thank you!
 Ram

On Mon, Apr 20, 2015 at 8:06 AM, Vishwas Babu A J <notifications@github.com>
wrote:

> In app/views/clients/clients.html
> <https://github.com/openMF/community-app/pull/1337#discussion_r28661242>:
>
> > @@ -3,10 +3,16 @@
> >          <li class="active">{{'label.anchor.clients' | translate}}</li>
> >      </ul>
> >      <api-validate></api-validate>
> > -
> > -    <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="marginbottom0px form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
> > -    <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
> > -
> > +    <div class="col-sm-5">
> > +        <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
> > +    </div>
> > +        <form ng-submit="search()">
> > +            <div class="col-sm-4">
> > +            <input  data-ng-model="client.query" type="text" placeholder="Search Client with Name/ExternalId/Identifier/Acc.No" value="{{query}}" class="form-control"/>
> > +            </div>
> > +            <button id="query" type="search" class="btn btn-primary" has-permission='READ_CLIENT'>Search Client</button>
> > +            <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
> > +        </form>
> >      <table class="table">
>
> Same comments would apply to other html pages
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/community-app/pull/1337/files#r28661242>.
>